tests/test-class-wpseo-metabox.php

```diff
@@ -91,6 +91,40 @@ public function test_column_heading_has_metadesc() {
 	}
 
 	/**
+	 * @covers WPSEO_Metabox::column_hidden
```

Comment:
I would also like to see a test if I decided to hide the seo title, but show the focuskeyword. So this means I would at least expect these tests:
- all hidden columns that stay hidden;
- all hidden columns that are not hidden;
- 1/2 columns that are hidden and the rest is not hidden.